src/components/Toast.tsx

```diff
@@ -37,20 +35,24 @@ const Toast = () => {
           left: [null, null, "32px"],
           right: [null, null, "32px"],
           top: "32px",
-        }
+        },
       } }
     >
       <Alert
         iconMapping={ {
           error: <CloseCircleFill />,
-          success: <CheckCircleIcon />
+          success: <CheckCircleIcon />,
         } }
         onClose={ handleClose }
         severity={ severity }
         sx={ {
           "&.MuiAlert-root": {
             ".MuiAlert-message": {
-              paddingRight: ["8px", "16px", "48px"]
+              paddingRight: [
+                theme.spacing(1),
```

Comment:
I think you forgot to remove the theme.spacing here. Not a big deal tho

Reply:
there are soooooo many ways to do the same thing lol i didnt know you could use the numbers just like that, i actually prefer the numbers as well.... will be changing it soon